PharoObjectModel/PharoObjectModel.pier

```diff
@@ -219,46 +219,50 @@ by instances of that class (and of its subclasses).
 !!!Class methods
 
 Class methods can be quite useful; browse ==Color class== for some good
-examples.You will see that there are two kinds of method defined on a class:
-those that create instances of the class, like ==Color class>>blue== and those
-that perform a utility function, like ==Color class>>showColorCube==. This is
+examples. You will see that there are two kinds of methods defined on a class:
+''instance creation methods'', like ==Color class>>blue==, and those that perform
+a utility function, like ==Dictionary class>>removeUnreferencedKeys==. This is
```

Comment:
This is strange to talk about Dictionary here because we mentioned Color just above.